src/Form/Form.tsx

```diff
@@ -1,9 +1,56 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
```

Comment:
Ooooo Gettin' Hooked, huh!? Wanna teach the hooks lesson this week? I'm kidding, but also totally not - if any of you want to try, I'd be there to support ya!

Reply:
Yes, hooks are awesome but they took some getting used to! Looking forward to your lesson this week :]